### PR TITLE
Freebsd 14 add

### PR DESCRIPTION
**Pull request by bcoca**
also remove 12.4

### CODE REVIEW

**Comment by mattclay** [comment]
Update https://github.com/ansible/spare-tire/blob/main/wheel_matrix.py first, and use that to generate the YAML file.

File: wheel_matrix.py [hunk 1]
```diff
@@ -19,6 +19,7 @@ class Package:
 def main() -> None:
     freebsd_pythons = {
         '13.2': ['3.9', '3.11'],
+        '14.0': ['3.11'],
```
Comment (mattclay) [inline wheel_matrix.py:22]: Wheels are needed for Python 3.9 also:

```suggestion
        '14.0': ['3.9', '3.11'],
```

**Review by mattclay — CHANGES_REQUESTED** [review]

File: wheel_matrix.py (hunk 1 above)
Comment (mattclay) [reply wheel_matrix.py:22]: For anyone wondering why, the reason is that we install packages in virtual environments too. So it's not enough for these to be available as OS packages for a given Python.

Comment (bcoca) [reply wheel_matrix.py:22]: i was looking to avoid all wheels for all versions, since 3.9 does not need cryptography

Comment (mattclay) [reply wheel_matrix.py:22]: Python 3.9 does need cryptography (see my previous comment).

Comment (mattclay) [reply wheel_matrix.py:22]: I've unresolved it, since Python 3.9 is needed.

Comment (bcoca) [reply wheel_matrix.py:22]: not sure what you mean, it is there, do you want me to remove 3.9?

Comment (bcoca) [reply wheel_matrix.py]: nvmd, i was not pushign to the PR branch

**Review by mattclay — APPROVED** [review]

**Pull request merged**